images/mongo/docker-entrypoint

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+exec "$@"
```

Comment:
this is already defined in the main entrypoints.sh file:
https://github.com/amazeeio/lagoon/blob/master/images/commons/lagoon/entrypoints.sh#L15

so we don't need that file at all :)